### ISSUE TITLE
How to apply threejs postprocessing in 3d-force-graph?

### ISSUE THREAD

**Issue by geohuz** [vasturiano/3d-force-graph#265]
In short, postprocessing in threejs is like apply multiple processing effect like adding a vignette, blurring applying a bloom etc and finally rendering the result to the canvas. And there is a EffectCompose function to compose all the processing 'passes' like below:

```
    const compose = new EffectComposer(renderer)
    composer.addPass(new RenderPass(scene, camera))
    const bloomPass = new BloomPass(params...)
    composer.addPass(bloomPass)
```

the last thing we need to do is to use **EffectComposer.render** instead of **WebGLRenderer.render** ,

```
    function render() {
      // the normal renderer is being replace by composer
     // renderer.render(scene, camera); 
     composer.render();
     requestAnimationFrame(render);
```

So I'm wondering is there something we can 'replace' the Graph.renderer() with the composer? I saw there is a renderer() function but I'm not sure how to replace it.  I tried the following code but I can't see any effect being applied:

```
          var composer = new EffectComposer( Graph.renderer());
          var renderScene = new RenderPass(Graph.scene(), Graph.camera())
          composer.addPass( renderScene );
          composer.addPass( bloomPass );
          composer.setSize(window.innerWidth, window.innerHeight)
          Graph.renderer(()=>composer.renderer()) 
```

**Comment by geohuz** [comment]
I actually found a way to get the composer work, basically change your /node_modules/3d-force-graph/dist/3d-force-graph.module.js, at line 158 adding postProcessingComposer, like below:

```
var linkedRenderObjsProps = Object.assign.apply(Object, _toConsumableArray(['width', 'height', 'backgroundColor', 'showNavInfo', 'enablePointerInteraction', 'postProcessingComposer'].map(function (p) {
  return _defineProperty({}, p, bindRenderObjs.linkProp(p));
})));
```

and in your code just do this:

```
Graph.postProcessingComposer(composer)
```

I not sure why the composer doesn't get exposed, may be there is a performance issue.

**Comment by vasturiano** [comment]
Thanks for reaching out @geohuz. The method `postProcessingComposer` is now exposed in the latest published version (`1.57.0`).
